src/reducers/__tests__/__snapshots__/reducers-test.js.snap

```diff
@@ -26,10 +26,12 @@ Object {
   "notifications": Object {
     "general": true,
   },
-  "projectList": Object {},
-  "projectWorkflows": Object {},
+  "projectList": Array [],
+  "projectListHolding": Array [],
```

Comment:
It's just for UI.  While the projectList is being fetched it has a lot of promises and if I were directly modifying projectList it would remove stuff and then put it back, resulting in a jumpiness.  This way I just wait until everything's fetched and then reassign the projectList at once then.